packages/dev/gui/src/2D/controls/container.ts

```diff
@@ -208,6 +209,9 @@ export class Container extends Control {
         this._reOrderControl(control);
 
         this._markAsDirty();
+
+        this.onControlAddedOrRemovedObservable.notifyObservers(control);

```

Comment:
I would prefer having OnAdd and another OnRemoved

Reply:
Changed to OnAdd and OnRemoved!